homeassistant/components/bkk/__init__.py

```diff
@@ -0,0 +1 @@
+"""Example Load Platform integration."""
```

Comment:
Please adjust this to fit your integration.

Reply:
done